invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/components/CommunitySelectionModal/CommunitySelectionFooter.js

```diff
@@ -17,7 +17,7 @@ export const CommunitySelectionFooter = () => {
           <p>
             <Trans>
               Did not find a community that fits you? Upload without a community or{" "}
-              <a href="/communities/new">create your own.</a>
+              <a href="/communities-new">create your own.</a>
```

Comment:
also all these places should be changed to utilize the config rather the hardcoded values

Reply:
It's a javascript file, I'm not sure if we are already injecting config?